src/TwitterAds/Analytics.php

```diff
@@ -41,23 +68,30 @@ public static function all_stats(Account $account, $ids, $metricGroups, $params
         $granularity = isset($params['granularity']) ? $params['granularity'] : Enumerations::GRANULARITY_HOUR;
         $placement = isset($params['placement']) ? $params['placement'] : Enumerations::PLACEMENT_ALL_ON_TWITTER;
         $segmentationType = isset($params['segmentation_type']) ? $params['segmentation_type'] : null;
+        // added this in order to get the campaing level stats - this NOT recomeded
+        $entity = isset($params['entity']) ? $params['entity'] : static::ENTITY ;
 
         $params = [
             'metric_groups' => implode(",", $metricGroups),
             'start_time' => $startTime->format('c'),
             'end_time' => $endTime->format('c'),
             'granularity' => $granularity,
-            'entity' => static::ENTITY,
```

Comment:
Instead of changing this, you can do in `Campaign` --> `class Campaign extends Analytics`, its cleaner.